compiler/luci-interpreter/src/core/KernelParams.h

```diff
@@ -123,6 +123,12 @@ struct StridedSliceParams
   int32_t shrink_axis_mask;
 };
 
+struct SqueezeParams
+{
+  int8_t squeeze_dims_count;
+  int32_t squeeze_dims[4];
```

Comment:
Why not use vector?

Reply:
Oh, sorry, I refer some wrong referred source. I will change into a int vector.

Reply:
It is indeed a simple array in TFLite, but we don't need to copy them.

Reply:
yes. I saw wrong reference. sorry for confused.